### ISSUE TITLE
[RETEST] Mobile A11Y: focus stays trapped at the bottom of the page after clicking "View Cart" button in the Add-to-cart modal

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13765]
**Additional context**

- Fix this ticket only after the whole Screen Reader milestone for desktop is completed. 

**Describe the bug**

-  Add-to-cart modal, if the "View Cart" button is clicked on, the focus stays trapped at the bottom of the page and doesn't read the new page title.

**To Reproduce: Watch Test Videos**

- Android Talkback test video
https://video.sap.com/media/t/1_1gtfpn6o
min 0:13:00
  
- Iphone VoiceOver test video
https://video.sap.com/media/t/1_qa70rxtl
min 0:04:54

**Expected behavior**
- Page title gets vocalized when the user lands on a new page.

**Comment by pawelfras** [comment]
Couldn't reproduce on iOS and Android phone for latest develop version.
